qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -133,6 +133,7 @@ def _vf2_match_not_found(property_set):
             call_limit=int(5e4),  # Set call limit to ~100ms with rustworkx 0.10.2
             properties=backend_properties,
             target=target,
+            max_trials=int(2500),  # Limits layout scoring to < 600ms on ~400 qubit devices
```

Comment:
What about `VF2PostLayout`, shouldn't we be setting this there too?

Reply:
Added in 9c0e5e8 😄